coral-component-masonry/examples/index.html

```diff
@@ -221,8 +221,25 @@ <h2 class="coral-Heading--M">Usage notes</h2>
         </div>
         <script>
           window.addEventListener('load', function() {
+            let uid = 0;
+            function getUID() {
```

Comment:
Please use `commons.getUID` to get unique id. https://github.com/adobe/coral-spectrum/blob/master/coral-utils/src/scripts/Commons.js#L555

Reply:
`commons` not defined when checking examples page only for masonry when developing locally. Hence resolving this.